tests/integration_test/backend_auth_using_imds_test.go

```diff
@@ -48,6 +48,7 @@ func TestBackendAuthWithImdsIdToken(t *testing.T) {
 		desc     string
 		method   string
 		path     string
+		headers  map[string]string
 		message  string
 		wantResp string
```

Comment:
add integration like you did in ESP for the combinations of cases:
- if the request originally has "Authorization"
- if the "Authorization" token is needed to be overwritten.

Reply:
Done, add two cases in backend_auth_disable_auth_test.go